docs/middleware.md

```diff
@@ -44,12 +44,21 @@ A very simple middleware that just logs the invocation of actions will look like
 @example
 ```typescript
 const store = SomeStore.create()
-const disposer = addMiddleWare(store, (call, next) => {
+const disposer = addMiddleWare(store, (call, next, abort) => {
   console.log(`action ${call.name} was invoked`)
   return next(call) // runs the next middleware (or the implementation of the targeted action if there is no middleware to run left)
 })
 ```
 
+@example
+```typescript
+const store = SomeStore.create()
+const disposer = addMiddleWare(store, (call, next, abort) => {
+  console.log(`action ${call.name} was invoked`)
+  return abort('value') // aborts running the middlewares and returns the 'value' instead. Note that the underlying action won't be invoked either.
+})
+```
+
 _Note: for middleware, it is extremely important that `next(action)` is called for asynchronous actions (`call.type !== "action"`), otherwise the generator will remain in an unfinished state forever_
```

Comment:
I think this note can be removed?

Also, can you add a test to verify that having an aborting middleware does properly finish the generator when using asynchronous actions?